src/com/lftechnology/java/training/sanish/numbergame/NumberGame.java

```diff
@@ -0,0 +1,46 @@
+package com.lftechnology.java.training.sanish.numbergame;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class NumberGame {
+	private static final Logger LOGGER = Logger.getLogger(NumberGame.class
+			.getName());
+
+	public static void main(String[] args) {
+		try (Scanner scanner = new Scanner(System.in)) {
+			startGuessNumber(scanner);
+		} catch (Exception e) {
+			LOGGER.log(Level.WARNING, "Exception Message : {0}",
+					new Object[] { e.getMessage() });
+		}
+	}
+
+	/**
+	 * Start guess number game
+	 * 
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+	 */
+	public static void startGuessNumber(Scanner scanner) {
+		GuessNumberGame guessNumberGame = new GuessNumberGame();
+		int guessNumber;
+		LOGGER.log(Level.INFO, "Enter a guess number (1-20):");
```

Comment:
what if the errors occurs(like number is graater than max_limit ). Then why not again promt the user to enter the guess number again..Since the user may not guess by himself...what to do next